src/App.css

```diff
@@ -1,38 +1,69 @@
-.App {
-  text-align: center;
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+.nav {
+  padding-top: 2%;
+  height: 95.8vh;
+  background-color: white;
+  width: 150px;
+  border-right: 0.5px solid rgb(213, 213, 213);
 }
 
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+.logo {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.logo img {
+  width: 100px;
 }
 
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
+.navigation {
+  margin-top: 50px;
   display: flex;
   flex-direction: column;
   align-items: center;
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
+  justify-content: space-between;
```

Comment:
## Optional 

- I think you can have a name for a style you might constantly use e.g. flex-class. once you add the class a feature it immediately takes the style of the class.  I believe this will make the style really simple and neat